src/styles/body.scss

```diff
@@ -0,0 +1,85 @@
+@use "sass:map";
+
+$starting-day: 'fri';
+$month-length: 29;
+$block-size: 100px;
+$columns: 7;
+$gap: 1px;
+$padding: 10px;
+$border-color: darkgrey;
+$background-color: #eee;
+$background-hover-color: pink;
+$font-color: black;
+$month-days: (
+  'mon':1,
+  'tue':2,
+  'wed':3,
+  'thu':4,
+  'fri':5,
+  'sat':6,
+  'sun':7);
+$get-day: map.get($month-days, $starting-day);
+
+
+.container {
+  font-family: Arial, Helvetica, sans-serif;
+  display: flex;
+  flex-wrap: wrap;
+  gap: $gap;
+  max-width: calc(#{$columns} * #{$block-size} + (#{$columns} - 1) * #{$gap} + #{$padding * 2});
+  margin: 50px auto;
+}
+
+.block {
+  width: $block-size;
+  height: $block-size;
+  background-color: $background-color;
+  color: $font-color;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  font-size: 30px;
+  border: 1px solid $border-color;
+  box-sizing: border-box;
```

Comment:
would be better to `*` selector
```suggestion
```

Reply:
I didnt quite understand what you mean by that comment :/

Reply:
Just move it to the `*` selector)

Reply:
Done! Please check my new commit when you can :)